### ISSUE TITLE
[feature request] material you theme

### ISSUE THREAD

**Issue by zoroya1** [SankethBK/diaryvault#256]
Material you theme will look good 👍 
Also random quotes while opening the app or at home screen will light up the user's mood

**Comment by SankethBK** [comment]
Hi, we are also planning to add a material theme. Quotes is a nice idea, i just need to think where to fit it in the UI.

**Comment by njouud** [comment]
Hello, are there plans or any guidelines for adding the quotes? I am interested in adding that feature if possible.

**Comment by SankethBK** [comment]
Hi @njouud , you can refer [themes wiki](https://github.com/SankethBK/diaryvault/wiki/Themes)

**Comment by SankethBK** [comment]
@njouud I don't have anything in mind right now,, if you have have any can you post a rough sketch or worefrt

**Comment by njouud** [comment]
Yeah of course, this is a mockup of a potential implementation, the user would view a screen with a randomized quote and the diaryvault logo which would disappear after 5 seconds or so, also works as a welcome screen. Let me know your thoughts! thanks 
<img src="https://github.com/user-attachments/assets/737245ee-7210-40b9-b7d8-558e024054bf" alt="Alt Text" width="192.25" height="431.875">

**Comment by SankethBK** [comment]
Looks good, is it home screen or splash screen

**Comment by njouud** [comment]
It would be like a splash screen when the app is opened if that sounds good

**Comment by SankethBK** [comment]
yes that's cool

**Comment by basings** [comment]
@SankethBK are there still plans to include a material you theme?
